rt_bi_behavior:
  dot_renderer:
    ros__parameters:
      template_dir: "config"
      template_file: "graphviz.html"
      dot_string: ""
  ba1:
    ros__parameters:
      render: True
      grammar_dir: "config"
      grammar_file: "transition.lark"
      states:
        - "Q0"
        - "Q1"
        - "Q2"
      transitions_from:
        - "Q0"
        - "Q1"
      transitions_predicate:
        - '[material.name == "street"] OR [material.name == "bridge"]'
        - '[name == "bridge-is-moving"] AND [name == "storage"]'
      transitions_to:
        - "Q1"
        - "Q2"
      start: "Q0"
      accepting:
        - "Q2"
